moonraker/components/authorization.py

```diff
@@ -383,14 +418,32 @@ def _login_jwt_user(self,
             action = "user_created"
         else:
             if username not in self.users:
-                raise self.server.error(f"Unregistered User: {username}")
+                if self.use_ldap:
```

Comment:
This is a great idea. However, I would like to have the feature that LDAP authentication can be done without a database account. Otherwise, moonraker would open the door every time there are no database accounts. Even if an LDAP configuration is provided. Please correct me if I am wrong. 

And I like your code suggestion.